binaries/scheduler/main.go

```diff
@@ -17,21 +19,22 @@ func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	parser := config.DefaultParser()
-	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
-	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
-	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	env, o := config.Default()
```

Comment:
why is env being determined by the scheduler config?  Shouldn't it be determined by a flag passed in as part of the aurora config? 

Ideally we want different config for local, devel, staging, and prod